### PR TITLE
Add Pitivi

### PR DESCRIPTION
**Pull request by aleb**

### CODE REVIEW

File: _data/projects/pitivi.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- python
+- video
+- video-editing
+- gstreamer
+- gtk
+- gui
+- gnome
+upforgrabs:
+  name: 'Newcomers'
+  link: 'https://gitlab.gnome.org/GNOME/pitivi/issues?label_name%5B%5D=4.+Newcomers'
```
Comment (ritwik12) [inline _data/projects/pitivi.yml]: @aleb Please remove apostrophes from both lines.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/pitivi.yml (hunk 1 above)
Comment (aleb) [reply _data/projects/pitivi.yml]: done

**Review by shiftkey — APPROVED** [review]
Thanks @aleb!

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@aleb Thanks!!

**Pull request merged**